CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-# dbt_quickbooks version.version
+# dbt_quickbooks v0.17.2
+[PR #155](https://github.com/fivetran/dbt_quickbooks/pull/155) introduces the following updates:
+
+## Bug Fixes
+- Modified `int_quickbooks__bill_payment_double_entry model` to ensuree bill payment accuracy by updating the `converted_amount` logic:
+  - Utilized the line-level `amount` from bill payment lines instead of `total_amount` from bill payments.
```

Comment:
Can you add _why_ you applied this change.

Reply:
Added.